x/wasm/types/connector.go

```diff
@@ -36,7 +36,9 @@ func ParseEvents(
 
 		// Deprecated: from_contract
 		sdkEvent.Type = EventTypeFromContract
-		sdkEvents = sdkEvents.AppendEvent(*sdkEvent)
+		if sdkEvent != nil {
```

Comment:
If sdkEvent is nil, this code will panic on lines 35 and 38, before it gets here. The problem is that buildEvent can return a nil *sdk.Event _and_ a nil error. That should not happen. Either have buildEvent return an error if len(attributes) == 0, or check len(attributes) before calling buildEvent.

Reply:
+ in line 29, len(attributes) is checked